src/mds/MDSAuthCaps.h

```diff
@@ -276,8 +276,7 @@ class MDSAuthCaps
     }
 
     for (const MDSCapGrant &g : grants) {
-      if (g.match.fs_name == fs_name || g.match.fs_name.empty() ||
-	  g.match.fs_name == "*") {
+      if (fs_name_match(g.match.fs_name, fs_name)) {
```

Comment:
I find it more readable when function members are named -

```suggestion
      if (does_fsname_match(g.match.fs_name, fs_name)) {
```

Even better if we make it method in `MDSCapMatch`, this way it will take less typing effort and will be easier to  use -

```suggestion
      if (g.match.does_fsname_match(fs_name)) {
```

Reply:
Sure, done.